server/src/models/session.js

```diff
@@ -13,7 +13,18 @@ const Session = model(
         ref: 'Quest',
       },
       containerId: String,
+      status: {
+        type: String,
+        enum: ['active', 'finished', 'inactive'],
+        required: true,
+        default: 'active',
+      },
```

Comment:
Does this enum store all the session seperately in active, finished and inactive?
How to define finished session?

Reply:
Yes, every session has its own status, which is either active, finished, or inactive